### DISCUSSION TITLE
Is it possible to toggle/cycle stashes directionally?

### DISCUSSION THREAD

**Discussion by vlnn** [rxhanson/RectanglePro-Community#451] [Announcements]
I may be blind, but can't find a way to toggle everything stashed on the right, hide it, then toggle or cycle everything stashed on the left, and have a separate shortcuts for these actions? Or this is already somewhere in the system and I just cant see it?

**Comment by rxhanson** [comment]
This is not currently in the app. It's on my backlog, though, and I'll add your +1 to it.
